docs/user/visualize.asciidoc

```diff
@@ -42,6 +42,7 @@ To create a visualization:
 locations.
 * *Time Series*
 [horizontal]
+<<lens,Lens>>:: Create and build basic visualizations using time series data fields.
```

Comment:
I disagree...for now. When we merge the greater Visualize changes, this will fit nicely under `Most frequently used visualization`. But for now, it still fits under the Basic charts category.

Reply:
Lens is not a time series visualization tool, and it uses different aggregations/UI than the other items in "Most frequently used." Maybe that section is not named well enough